components/dashboard/filters/HostedAccountFilter.tsx

```diff
@@ -61,7 +59,7 @@ export const AccountRenderer = ({
   return (
     <AccountHoverCard
       account={data.account}
-      {...(inOptionsList && { hoverCardContentProps: { side: 'right', sideOffset: 24 } })}
+      {...(props.inOptionsList && { hoverCardContentProps: { side: 'right', sideOffset: 24 } })}
```

Comment:
Fixes an issue where only the slug and not the full data (name + avatar) was used to display the current filter value